astroquery/mast/core.py

```diff
@@ -1558,19 +1860,31 @@ def query_criteria_async(self, catalog, pagesize=None, page=None, **criteria):
             only one wildcarded value can be processed per criterion.
             RA and Dec must be given in decimal degrees, and datetimes in MJD.
             For example: filters=["FUV","NUV"],proposal_pi="Ost*",t_max=[52264.4586,54452.8914]
+            For catalogs available through Catalogs.MAST (PanSTARRS), the Column Name is the keyword with the argument
+            should be either an acceptable value for that parameter, or a list consisting values or  tuples of decorator,
```

Comment:
Something is slightly wrong with this sentence I think? 
"the Column Name is the keyword with the argument should be either an acceptable value for that parameter" -> "the Column Name is the keyword, and the argument should be either an acceptable value for that parameter"